### PR TITLE
Change error message for GetRequiredService

### PR DESCRIPTION
**Pull request by sungam3r**
Inspired by #3374

### CODE REVIEW

File: src/GraphQL/Utilities/ServiceProviderExtensions.cs [hunk 1]
```diff
@@ -28,7 +28,7 @@ public static object GetRequiredService(this IServiceProvider provider, Type ser
             if (service != null)
                 return service;
 
-            throw new InvalidOperationException($"Required service for type {serviceType} not found");
+            throw new InvalidOperationException($"No service for type '{serviceType}' has been registered.");
```
Comment (sungam3r) [inline src/GraphQL/Utilities/ServiceProviderExtensions.cs:31]: Copied from MS implementation.

Comment (Shane32) [reply src/GraphQL/Utilities/ServiceProviderExtensions.cs:31]: I suggest we use the friendly name extension method we have for better support of generic types

Comment (Shane32) [reply src/GraphQL/Utilities/ServiceProviderExtensions.cs:31]:
```suggestion
            throw new InvalidOperationException($"No service for type '{serviceType.GetFriendlyName()}' has been registered.");
```

Comment (sungam3r) [reply src/GraphQL/Utilities/ServiceProviderExtensions.cs]: ok

**Review by Shane32 — APPROVED** [review]

**Pull request merged**